src/PolynomialSolutions.jl

```diff
@@ -454,6 +512,114 @@ function solve_laplace(Q::Polynomial{N,T}) where {N,T}
     return P
 end
 
+"""
+    solve_anisotropic_laplace(A::AbstractMatrix{T}, Q::Polynomial)
+
+Return a polynomial `P` satisfying the anisotropic Laplace equation `∇ ⋅ (A ∇P) = Q`, `A` an invertible
+matrix. `Q` is required to be homogeneous. Inverse is anisotropic_laplacian.``
+
+# Examples
+
+```jldoctest
+using StaticArrays
+A = SMatrix{2,2,Rational{Int64}}(2 // 1, 1 // 1, 1 // 1, 3 // 1)
+Q = Polynomial([(1, 1) => 2 // 1])
+P = solve_anisotropic_laplace(A, Q)
+
+# output
+
+-3//400x⁴ + 11//100x³y + 11//150xy³ - 2//25x²y² - 1//300y⁴
+```
+"""
+function solve_anisotropic_laplace(A::AbstractMatrix{T}, Q::Polynomial{N,T}) where {N,T}
+    @assert LinearAlgebra.checksquare(A) == N
+    @assert A == transpose(A) "anisotropic tensor must be symmetric"
+    @assert is_homogeneous(Q) "source term `Q` must be a homogeneous polynomial"
+
+    n = degree(Q)
+    γ = (k, p) -> big(2 * (k + 1) * (2k + 2p + N)) # γₖᵖ
+    cₖ = 1 // γ(0, n) # c₀
+    P = cₖ * multiply_by_anisotropic_r(A, deepcopy(Q), 2)
+    ΔᵏQ = deepcopy(Q)
+    m = floor(Int, n / 2)
+    for k in 1:m
+        cₖ = -cₖ / (γ(k, n - 2k))
+        ΔᵏQ = anisotropic_laplacian(A, ΔᵏQ)
+        ΔP = cₖ * (multiply_by_anisotropic_r(A, ΔᵏQ, 2k + 2))
+        P = P + ΔP
+    end
+    #return P
+    return convert_coefs(P, T)
```

Comment:
I am afraid this may be too restrictive. What if I pass `Q` which is a polynomial with an integer coefficient? Convert should fail then.

Reply:
I wish I knew a good way to handle this. We can warn on passing in an `Integer` typed object that this will likely not succeed? Other than that I don't know the appropriate julia-foo.

Reply:
`@warn T <: Integer "This routine has a converted-to return type identical to input; integer input likely to fail"` ?

Reply:
This isn't great either.